src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenuCard/FeatureStrategyMenuCard.tsx

```diff
@@ -0,0 +1,47 @@
+import { IStrategy } from 'interfaces/strategy';
+import { Link } from 'react-router-dom';
+import { useStyles } from './FeatureStrategyMenuCard.styles';
+import {
+    getFeatureStrategyIcon,
+    formatStrategyName,
+} from 'utils/strategy-names';
+import { formatCreateStrategyPath } from '../../FeatureStrategyCreate/FeatureStrategyCreate';
+
+interface IFeatureStrategyMenuCardProps {
+    projectId: string;
+    featureId: string;
+    environmentId: string;
+    strategy: IStrategy;
+}
+
+export const FeatureStrategyMenuCard = ({
+    projectId,
+    featureId,
+    environmentId,
+    strategy,
+}: IFeatureStrategyMenuCardProps) => {
+    const styles = useStyles();
+    const StrategyIcon = getFeatureStrategyIcon(strategy.name);
+    const strategyName = formatStrategyName(strategy.name);
+
+    const createStrategyPath = formatCreateStrategyPath(
+        projectId,
+        featureId,
+        environmentId,
+        strategy.name
+    );
+
+    return (
+        <Link to={createStrategyPath} className={styles.card}>
+            <div className={styles.icon}>
+                <StrategyIcon aria-hidden />
+            </div>
+            <div>
+                <div className={styles.name}>
```

Comment:
We should probably use `<StringTruncator />` here: 
<img width="363" alt="Skjermbilde 2022-03-09 kl  11 36 17" src="https://user-images.githubusercontent.com/16081982/157426406-f3b55c41-30cf-47bd-9fd2-2ed3f99b6509.png">